configure.ac

```diff
@@ -5,6 +5,12 @@ AC_PROG_CC
 
 AC_PATH_PROG(APXS, 'apxs')
 
+if test x"$APXS" = x"" ; then
+  AC_CHECK_FILE([/etc/redhat-release],
```

Comment:
as long as it's reliable; I still have fear of voodoo that you probably don't.  fine with me then.